### ISSUE TITLE
Switch from our own implementation of MultiHeadAttention to the one created by tensorflow 

### ISSUE THREAD

**Issue by ChaoPang** [cumc-dbmi/cehrbert#36]
- Upload Tensorflow to 2.12 so we can use TensorFlow's version of MultiHeadAttention
- Replace our own implementation of [MultiHeadAttention](https://github.com/cumc-dbmi/cehr-bert/blob/8862fd6a1f026839d301625fb30a1ba7e19a4672/models/custom_layers.py#L81) with the Tensorflow version of [MultiHeadAttention](https://www.tensorflow.org/api_docs/python/tf/keras/layers/MultiHeadAttention)
- Toggle `mask` at https://github.com/cumc-dbmi/cehr-bert/blob/8862fd6a1f026839d301625fb30a1ba7e19a4672/data_generators/learning_objective.py#L281. Because `mask=1` indicates attention whereas `mask=0` represents no attention
